train.py

```diff
@@ -339,27 +418,38 @@ def main():
                 loss, kl = objective(log_probs=log_probs, experience=exp)
```

Comment:
if we use model.forward() .. probably should do this here too, but I could also do that later.

Reply:
I added back forward in the objective call